app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_cp = command.split()
+    if len(command_cp) != 3 or command_cp[0] != "cp":
+        return
```

Comment:
try not to use blank return here, make a backward condition
You can do it with this check together `if source == destination`

Reply:
You still have blank returns. Fix throughout the code